### PR TITLE
Updating reference docs to v9

### PR DESCRIPTION
**Pull request by crgrieve**
Making the reference page v9 by default but adding a v8 version and some links to v8 versions where relevant.

### CODE REVIEW

File: Reference/Notifications/index.md [hunk 1]
```diff
@@ -160,4 +160,4 @@ Useful for manipulating the model before it is sent to an editor in the backoffi
 
 # Creating and publishing your own custom notifications
 
-Umbraco uses notifications to allow people to hook in various workflow processes, but the notification pattern is also extendible, allowing you to create your own custom notification and publishing them, allowing other people to hook into your processes, this can be very useful when for instance creating packages. For more information on how you create and publish your own notifications see the [creating and publishing notifications](Creating-And-Publishing-Notifications.md) article.
\ No newline at end of file
+Umbraco uses notifications to allow people to hook in various workflow processes, but the notification pattern is also extendible, allowing you to create your own custom notification and publishing them, allowing other people to hook into your processes, this can be very useful when for instance creating packages. For more information on how you create and publish your own notifications see the [creating and publishing notifications](Creating-And-Publishing-Notifications.md) article.
```
Comment (marcemarc) [inline Reference/Notifications/index.md:163]:
```suggestion
Umbraco uses notifications to allow people to hook into various workflow processes. This notification pattern is extensible, allowing you to create and publish your own custom notifications, thus allowing other people to observe and hook into your custom processes. This approach can be very useful when creating Umbraco packages. For more information on how you create and publish your own notifications see the [creating and publishing notifications](Creating-And-Publishing-Notifications.md) article.
```

**Review by marcemarc — COMMENTED** [review]
Thanks Carole! all great, but in reviewing your changes I read that existing opening paragraph... and it felt a bit disjointed to me, I've made some suggestions... does it make a difference to its readability?

**Comment by crgrieve** [comment]
yeah, that makes more sense. Thanks @marcemarc!

**Comment by marcemarc** [comment]
thanks @crgrieve sorry for the delay! - merging this in! #h5yr

**Pull request merged**